services/entryvalidateService.js

```diff
@@ -0,0 +1,81 @@
+const sensorCachingService = require('./sensorCachingService');
+
+/**
+ * Checks the validation of the entry
+ * @author Onur Ereren
+ * @param {Sensor Object} entry 
+ * @returns {boolean} Returns true if the entry is correct otherwise false.
+ */
+function validateEntry(entry){
+
+    return validateTemperature(entry) && validateTemperatureWithLastEntry(entry) && validateTemperatureWithHighestandLowest(entry)
+}
+
+
+/**
+ * Checks the temperature of the entry.
+ * @author Onur Ereren
+ * @param {Sensor Object} entry 
+ * @returns {boolean} Returns true if the entry is correct otherwise false.
+ */
+function validateTemperature(entry) {
+    if (entry.parameter == 'temperature') {
+        if (!(entry.value > 15 && entry.value < 45)) {
+            return false;
+        }
```

Comment:
```suggestion
        return entry.value > 15 && entry.value < 45
```
simplify the if statement and remove the negation.
instead of "if this is not the case return false" just say "return wheter this is true or not"